README.md

```diff
@@ -60,6 +60,7 @@
 - [Ory Hydra](https://github.com/ory/hydra) - Open Source OpenID Certified™ OpenID Connect and OAuth Provider.
 - [panva/node-oidc-provider](https://github.com/panva/node-oidc-provider) - Open Source and certified OpenID Connect provider implementation in Node.js with FAPI 1.0 and FAPI 2.0 support.
 - [PingFederate](https://www.pingidentity.com/en/platform/capabilities/authentication-authority/pingfederate.html) - Federation server that provides secure single sign-on, API security and provisioning for enterprise customers, partners, and employees.
+- [Scalekit](https://www.scalekit.com) - Implements SAML and OIDC-based SSO and SCIM provisioning on top of your existing auth systems.
```

Comment:
The list focuses on OpenID Connect, and developers typically refer to it when they're looking for OIDC-related solutions. Could you emphasize more on your OP (OpenID Provider) implementation and/or how OIDC fits into your product?

Also, I’m not sure why SCIM and SAML are mentioned in connection with your OIDC implementation here. Could you clarify the link?